components/reddit/sources/new-comments-by-user/new-comments-by-user.js

```diff
@@ -0,0 +1,108 @@
+const reddit = require("../../reddit.app.js");
+module.exports = {
+  key: "new-comments-by-user",
+  name: "New comments by user",
+  description: "Emits an event each time a user posts a new comment.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    reddit,
+    username: { propDefinition: [reddit, "username"] },
+    context: {
+      type: "integer",
+      label: "context",
+      description: "an integer between 2 and 10",
+    },
+    t: {
+      type: "string",
+      label: "t",
+      description: "one of (hour, day, week, month, year, all)",
+      options: ["hour", "day", "week", "month", "year", "all"],
+    },
+    sr_detail: {
+      type: "boolean",
+      label: "Include Subreddit details?",
+      description:
+        "If set to true, includes details on the subreddit in the emitted event.",
+      default: false,
+    },
+    timer: {
+      label: "Polling schedule",
+      description:
+        "Pipedream polls Reddit on this schedule for new comments by the prop indicated user.",
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 10, // by default, run every 10 minute.
```

Comment:
I didn't catch this before, but we typically run these every 15 minutes by default.

Reply:
MM. I think at one point you suggested 10 minutes by default? Anyway, I made them all 15 minutes default now.